sled-agent/src/long_running_tasks.rs

```diff
@@ -73,46 +65,51 @@ pub async fn spawn_all_longrunning_tasks(
     oneshot::Sender<ServiceManager>,
 ) {
     let storage_key_requester = spawn_key_manager(log);
-    let mut storage_manager =
-        spawn_storage_manager(log, storage_key_requester.clone());
 
-    let storage_monitor_handle =
-        spawn_storage_monitor(log, storage_manager.clone());
+    let time_sync_config = if let Some(true) = config.skip_timesync {
+        TimeSyncConfig::Skip
+    } else {
+        TimeSyncConfig::Normal
+    };
+    let mut config_reconciler = ConfigReconcilerHandle::new(
```

Comment:
Did I miss where we call `spawn_reconciliation_task`, or is it not here yet?

Reply:
It can't happen this early in the process; it happens during `SledAgent::new()` (after we've gotten our config from RSS or the ledger): https://github.com/oxidecomputer/omicron/blob/abd7542457c460f059506a095aec1a4d790f13aa/sled-agent/src/sled_agent.rs#L600-L610